cdap-api/src/main/java/co/cask/cdap/internal/workflow/ProgramWorkflowAction.java

```diff
@@ -81,14 +78,14 @@ public void initialize(WorkflowContext context) throws Exception {
   public void run() {
     try {
       LOG.info("Starting Program for workflow action: {}", programName);
-      RuntimeContext runtimeContext = programRunner.call();
+      programRunner.run();
 
       // TODO (terence) : Put something back to context.
 
       LOG.info("{} Program {} workflow action completed",
                programType != null ? programType.name() : null, programName);
     } catch (Exception e) {
-      LOG.info("Failed to execute {} Program {} in workflow: {}", 
+      LOG.info("Failed to execute {} Program {} in workflow: {}",
                programType != null ? programType.name() : null, programName, e);
```

Comment:
I think `LOG` supports null value, you don't need the `?:` (if you don't call `.name()`, which is the same as `enum.toString()`.